wiz-admission-controller/values.yaml

```diff
@@ -140,6 +140,8 @@ imageIntegrityWebhook:
   cacheImageDigestTTL: 1m # The TTL of the cache for the image digest. The cache is used to avoid querying the image registry for the same image digest multiple times.
   cacheImageVerifyTTL: 5m # The TTL of the cache for the image verification. The cache is used to avoid verifying the same image multiple times.
 
+  patchImageDigestAnnotation: true # Should add annotations to pod that mapping between tag->digest.
```

Comment:
Maybe "If true, annotate the pod with mapping between tag->digest." is clearer. Up to you.